### ISSUE TITLE
Redundant warning in dfm_select with glob-type dictionary?

### ISSUE THREAD

**Issue by thomasd2** [kbenoit/quanteda#775]
```r
dfm_inaug <- dfm(data_corpus_inaugural)
(mydict <- dictionary(list(test = c("educ*", "carer*"))))
## Dictionary object with 1 key entry.
## - test:
##   - educ*, carer*

dfm_select(dfm_inaug, features = c("educ*", "carer*"), valuetype = "glob")
## Document-feature matrix of: 58 documents, 4 features (88.4% sparse).

# warns of multi-word features                    
dfm_select(dfm_inaug, features = mydict, valuetype = "glob")
## Document-feature matrix of: 58 documents, 4 features (88.4% sparse).
## Warning message:
## In features2vector(as.list(features)) :
##   dfm_select(dfm_inaug, features = mydict, valuetype = "glob") does not support multi-word features
```

**Comment by kbenoit** [comment]
Thanks @thomasd2 - I modified this a bit to show the output, but also to use only data objects that are available in the package. It works, but the warning should not be issued in the second case.

**Comment by kbenoit** [comment]
Solved with cf3110f99e16fd1285ce78f8a73061bd270d48fd.
